src/unit-of-work/index.js

```diff
@@ -0,0 +1,480 @@
+import Utils from '../utils'
+import Async from '../request/async'
+import DataQueryBuilder from '../data/query-builder'
+
+import { OperationType, IsolationLevelEnum } from './constants'
+import { OpResult } from './op-result'
+import { OpResultValueReference } from './op-result-value-reference'
+
+function resolveTableName(obj) {
+  return Utils.getClassName(obj)
+}
```

Comment:
I suggest either to remove this method and use Utils.getClassName, or rename getClassName to resolveTableName everywhere and use Utils. resolveTableName

Reply:
remove the method